tests/console/docker.pm

```diff
@@ -33,72 +37,57 @@ sub run {
         die "Docker is not pre-installed." if script_run("zypper se -x --provides -i docker | grep docker");
     }
     else {
-        zypper_call("in docker");
+        zypper_call("in docker"); # docker package can be installed
```

Comment:
tidy won't allow that. And also I don't think this particular comment helps better to understand that zypper tries to install docker. That should be self-explanatory

Reply:
The comment states why. To follow my suggestion of grouping the steps needed. Do you suggest to put the comment on a different line? To put it before the line 35 (35 after my changes)

Reply:
Yes, put it in before. I think github calls it R40 for line 40 after changes corresponding to L36 for line 36 before your changes